readit/database.py

```diff
@@ -30,7 +31,15 @@ def init_db(self, cursor, db):
         """
 
         try:
-            self.db = sqlite3.connect('test.db')
+            path = os.path.expanduser('~') + "/.config/readit"
+            if not os.path.exists(path):
+                os.mkdir(path)
+        except OSError:
+            print('Error: Creating directory.' + path)
+
+        databasefile = path + "/bookmarks.db"
```

Comment:
`os.path.join` here as well.

Reply:
@bhavin192 thanks
I will make use of it. It's portable too.